database/factories/AvailabilityFactory.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/** @var \Illuminate\Database\Eloquent\Factory $factory */
+
+use App\Modules\Availability\Availability;
+use Faker\Generator as Faker;
+
+$factory->define(Availability::class, function (Faker $faker) {
+    $from = $faker->dateTimeThisYear;
+    $to = $faker->dateTimeInInterval($from, '+1 day');
+
+    return [
+        'user_id' => $faker->date(1000000, 9999999),
```

Comment:
This should actually create the users, especially if we are going to test with them.

Make sure to also import user class above

```suggestion
        'user_id' => factory(User::class),
```

Reply:
done